spec/acceptance/default_parameters_jira_10_spec.rb

```diff
@@ -0,0 +1,182 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+def on_supported_os
+  (fact('os.family') == 'Debian' and fact('os.release.major') == '22.04') or (fact('os.family') == 'RedHat' and fact('os.release.major') >= '8')
+end
+
+prepare = <<-EOS
+  package {'diffutils':
+    ensure  => installed
+  }
+  file_line{'enable show_diff':
+    path => '/etc/puppetlabs/puppet/puppet.conf',
+    line => 'show_diff = true'
+  }
+EOS
+
+pre = <<-EOS
+  if $facts['os']['family']  == 'RedHat' {
+    $java_package = 'java-17-openjdk'
+    $java_home = '/usr/lib/jvm/jre-17-openjdk'
+    $postgresql_version = '13'
+    $pgsql_package_name = 'postgresql-server'
+    $pgsql_data_dir = '/var/lib/pgsql'
+    $manage_dnf_module = $facts['os']['release']['major'] ? {
+      '8'       => true,
+      default => false # RHEL-9 has pgsql 13 as a default
+    }
+    $autoremove_command = 'dnf --exclude="systemd*" autoremove -y'
+  }
+  elsif $facts['os']['family']  == 'Debian' {
+    $java_package = 'openjdk-17-jre'
+    $java_home = '/usr/lib/jvm/java-17-openjdk-amd64'
+    $postgresql_version = '14'
+    $pgsql_package_name = "postgresql-${postgresql_version}"
+    $pgsql_data_dir = "/var/lib/postgresql/${postgresql_version}/main/"
+    $manage_dnf_module = false
+    $autoremove_command = 'apt autoremove -y'
+  }
+  $jira_install_dir = '/opt/jira/'
+  $postgres_service = 'postgresql'
+  $jira_service = 'jira'
+EOS
+
+pp = <<-EOS
+  # The output of `systemctl status postgresql` is non ascii which
```

Comment:
I have some questions here. Why does that even break the exec resource? And is that a patch that should be added to the postgresql module instead?

Reply:
This I've copied from the existing default_parameters_spec.rb. I don't know if it is still needed.